notebooks/MAST/exploring_jwst_transmission/comparing_spectral_types_jwst.ipynb

```diff
@@ -0,0 +1,7074 @@
+{
```

Comment:
<p>The first line in this cell should say:</p><blockquote>If you have questions, comments, or feedback about this notebook, please email&nbsp;<code>archive@stsci.edu</code>.</blockquote><p>You should not put your email on the notebook; you will get questions from users that MAST should be answering. Obviously this is a great notebook, and you should take pride in (and get credit for) your work. As an alternative, you could make your name a link to your website, if you have one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>